src/transformers/models/llama/convert_llama_weights_to_hf.py

```diff
@@ -332,7 +332,7 @@ def permute(w, n_heads, dim1=dim, dim2=dim):
 
 class Llama3Converter(TikTokenConverter):
     def __init__(self, vocab_file, special_tokens=None, instruct=False, model_max_length=None, **kwargs):
-        super().__init__(vocab_file, **kwargs)
+        super().__init__(vocab_file, additional_special_tokens=special_tokens, **kwargs)
```

Comment:
[Line 348 below](https://github.com/huggingface/transformers/pull/33538/files#diff-110a445233a8b15a0875998eeaf75cb8607b38a5daa736291dd058766879bbddR348) could possibly be removed, as the superclass will handle it.

Reply:
Yes agreed it can be removed! Thanks